src/Amadeus/Client/Struct/Ticket/UpdateRefund.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * amadeus-ws-client
+ *
+ * Copyright 2015 Amadeus Benelux NV
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @package Amadeus
+ * @license https://opensource.org/licenses/Apache-2.0 Apache 2.0
+ */
+
+namespace Amadeus\Client\Struct\Ticket;
+
+use Amadeus\Client\RequestOptions\TicketUpdateRefundOptions;
+use Amadeus\Client\Struct\BaseWsMessage;
+use Amadeus\Client\Struct\Ticket\UpdateRefund\ContractBundle;
+
+/**
+ * UpdateRefund request structure
+ *
+ * @package Amadeus\Client\Struct\Ticket
+ * @author Vladimir Kikot <shoxyoyo@gmail.com>
+ */
+class UpdateRefund extends BaseWsMessage
+{
+    /** @var ContractBundle $ContractBundle */
+    public $ContractBundle;
+
+    public $Version = "3.000";
```

Comment:
is this something that should be hardcoded? _(i have no background info on this specific message)_

Reply:
@DerMika this was obtained from the wsdl
```
<wsdl:message name="AMA_TicketUpdateRefundRQ_3.000">
```
Also I've noticed that refund-related classes `IgnoreRefund`, `InitRefund`, etc have similar fields
```
public $Version = "2.000";
```

Actually, I can do something like this
```
    /** @var ContractBundle $ContractBundle */
    public $ContractBundle;

    public $Version;

    /**
     * UpdateRefund constructor.
     *
     * @param TicketUpdateRefundOptions $options
     * @param string $version
     */
    public function __construct($options, $version = '3.000')
    {
        $this->ContractBundle = new ContractBundle($options->contractBundleId, $options->waiverCode);
        $this->Version = $version;
    }
```
so the version won't be hardcoded anymore. What would you say?

Reply:
Yeah i'd prefer that. But put the default value in the `TicketUpdateRefundOptions`. That way it can be changed if someone needs it to be changed, without having to adjust code in the library.

Reply:
I must have missed that during my review of `IgnoreRefund` and `InitRefund`...

Reply:
@DerMika makes sens, thanks. Please check https://github.com/amabnl/amadeus-ws-client/pull/407/commits/f0fa2d8a2789466d2436a5e848129cd471591597